### ISSUE TITLE
Image Quality

### ISSUE THREAD

**Issue by 350d** [Filipowicz251/mijia-1080P-hacks#52]
I have latest hack version with default settings (with `HTTP_ENABLED=0`) and this image quality:
![glancecam_cam3_2018-07-14_23-16-47](https://user-images.githubusercontent.com/429208/42730177-3410397c-87bc-11e8-8a63-21e2aadd9e2b.png)
Same lines/glitches with daylight conditions.
`Mi Home` has no live view and stuck on `Preparing to connect...1%`. I've tried on recovery FW version and with update to latest 159. On 159 RTSP doesn't work at all. 

LOG:
````Running Customm Script v1.8

Configuration:
  CLOUD_DISABLED=0
  CLOUD_STREAMING_DISABLED=0
  RTSP_ENABLED=1
  CONFIG_LINE=-b4096 -f20 -w1920 -h1080 -m4
  SSH_ROOT_PASS=qwerty123456
  DISABLED_OTA=0
  HTTP_ENABLED=0
  SAMBA_ENABLED=0
os-version
NAME=Buildroot
VERSION=2016.02-git-00292-gca900c9-dirty
ID=buildroot
VERSION_ID=2016.02-git
PRETTY_NAME="Buildroot 2016.02-git"
XIAOMI_VERSION=3.3.6_0099
XIAOMI_BUILDNO=312  
Forcing Mode 4
 Staring SSH Server
 Getting root access 
Changing password for root
New password: 
Retype password: 
Password for root changed by root

...Starting dropbear...
[205] Jan 01 08:00:09 Running in background

...Changing Language to English...
...Adding Protection to Change of Keys...
...Init GM Driver for 0099 version...
[581] Jan 01 08:00:28 Child connection from 192.168.1.7:63175
[581] Jul 15 11:16:00 Password auth succeeded for 'root' from 192.168.1.7:63175````

**Comment by ich008** [comment]
You should disable cloud streaming when enabling RTSP. The Cam cant handle two streams (it was something like bug that this sometimes worked in version 158)

**Comment by lars-sorensen** [comment]
I also experience this bad quality - and I have CLOUD_STREAMING_DISABLED=1

**Comment by matisaul** [comment]
I have the same issue, with version 99, and CLOUD_STREAMING_DISABLED=1

**Comment by easwee** [comment]
Can confirm I have this same issue - could be possibly related to running on 2.4ghz instead of 5ghz wifi? Although this is a dual-band camera - it should work.

**Comment by easwee** [comment]
> Can confirm I have this same issue - could be possibly related to running on 2.4ghz instead of 5ghz wifi? Although this is a dual-band camera - it should work.

Solved this by upgrading to latest firmware - now everyhitng works as expected.

**Comment by mgsqz** [comment]
> > Can confirm I have this same issue - could be possibly related to running on 2.4ghz instead of 5ghz wifi? Although this is a dual-band camera - it should work.
> 
> Solved this by upgrading to latest firmware - now everyhitng works as expected.

but It seems that unable to keep the hack after upgrading to latest firmware

**Comment by young-george** [comment]
I'm getting the same problem. Upgrading the firmware seems to remove the hack.

**Comment by willthrom** [comment]
When I started the development of this project Xiaomi followed a specific structure for the software. A  flawed one we exploited.  After that they tried to mitigate the effect, the hack used to bypass those mechanism, by restoring to the original 0099  and upgrading in a control manner.

However, they might have changed the process and I don´t have spare time to debug whatever they have done now. If someone wants to continue the development, they can contact me so I can speed up the handover.
